entries/h/home.saxo.json

```
@@ -0,0 +1,18 @@
+{
+    "Saxo Bank": {
+      "domain": "home.saxo",
+      "url": "https://www.home.saxo/",
+      "additional-domains": [
+        "saxotrader.com"
+      ],
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://www.help.saxo/hc/en-us/articles/360038307372",
+      "keywords": [
+        "banking",
+        "finance",
+        "investing"
```

It seems to me that they only offer investment accounts, but neither savings accounts or checking accounts, nor loans or insurance. I'd suggest to leave just `investing` as a keyword then in order to keep it in line with other entries.
```suggestion
        "investing"
```